docs/data/material/components/progress/ProgressMaterialYouPlayground.js

```diff
@@ -33,15 +47,16 @@ export default function ProgressMaterialYouPlayground() {
         },
         { propName: 'fourColor', knob: 'switch', defaultValue: false },
       ]}
-      renderDemo={(props) => (
-        <Box sx={{ width: 300 }}>
-          <LinearProgress
-            {...props}
-            value={value}
-            valueBuffer={valueBuffer + value}
-          />
-        </Box>
-      )}
+      renderDemo={({ type: progressType, ...props }) => {
+        const Progress =
+          progressType === 'CircularProgress' ? CircularProgress : LinearProgress;
+
+        return (
+          <Box sx={{ width: 300, display: 'flex', flexDirection: 'column', gap: 1 }}>
+            <Progress {...props} value={value} valueBuffer={valueBuffer + value} />
+          </Box>
```

Comment:
Would it be possible to center the Circular Progress one?

<img width="832" alt="Screenshot 2023-11-30 at 14 01 50" src="https://github.com/mui/material-ui/assets/16889233/00788477-98bf-48d5-bd23-bffd4bd32b17">